src/buildSlice.ts

```diff
@@ -236,11 +241,39 @@ export function buildSlice({
     },
   });
 
+  const configSlice = createSlice({
+    name: `${reducerPath}/config`,
+    initialState: {
+      online: true,
+      focused: true,
+      refetchOnMountOrArgChange: config.refetchOnMountOrArgChange,
+      refetchOnReconnect: config.refetchOnReconnect,
+      refetchOnFocus: config.refetchOnFocus,
```

Comment:
```suggestion
      ...config
```
is probably more future-proof and saves a few bytes